pkg/config/stepmeta.go

```diff
@@ -518,11 +518,10 @@ func ResolveMetadata(gitHubTokens map[string]string, metaDataResolver func() map
 	return metadata, nil
 }
 
-//ToDo: Enable this when the Volumes part is also implemented
-//func volumeMountsAsStringSlice(volumeMounts []VolumeMount) []string {
-//	e := []string{}
-//	for _, v := range volumeMounts {
-//		e = append(e, fmt.Sprintf("%v:%v", v.Name, v.MountPath))
-//	}
-//	return e
-//}
+func volumeMountsAsStringSlice(volumeMounts []VolumeMount) []string {
```

Comment:
Can we please add unit tests for this new functionality

Reply:
> Can we please add unit tests for this new functionality

Fair point. We will have a look.